store/tikv/region_cache.go

```diff
@@ -1114,6 +1212,12 @@ func (s *Store) initResolve(bo *Backoffer, c *RegionCache) (addr string, err err
 		}
 		addr = store.GetAddress()
 		s.addr = addr
+		for _, label := range store.Labels {
+			if label.Key == confTiFlash.LabelKey && label.Value == confTiFlash.LabelValue {
```

Comment:
Just curious, can `label.Value` be other values other than `confTiFlash.LabelValue`?

Reply:
It seems no other values except the `confTiFlash.LabelValue`. I'm not very sure, so I programmed this check with an iteration.